logic/round.js

```diff
@@ -222,7 +222,7 @@ class Round {
 						}
 					});
 				}
-				return Promise.resolve();
```

Comment:
?

Reply:
@yatki strongly suggested to return null in this cases instead of resolving promises. Test became unstable since we introduce this.